src/partials/50_section-products.html

```diff
@@ -72,7 +72,9 @@ <h5>Aenean Ru Bristique</h5>
           <div class="actions">
             <div class="outlines">
               <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
-              <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
+              <a href="#" class="btn-outline active"
+                ><i class="fas fa-exchange-alt"></i
```

Comment:
Wkradł Ci się tu drobny błąd w formatowaniu kodu. Wyglądałoby to zdecydowanie lepiej, gdyby cały link <a></a> był w jednej linii.

Reply:
fakt :) zabłąkany enter. poprawiam.